src/chromium/crBrowser.ts

```diff
@@ -50,6 +51,8 @@ export class CRBrowser extends BrowserBase {
     const browser = new CRBrowser(connection, options);
     browser._devtools = devtools;
     const session = connection.rootSession;
+    const version = await session.send('Browser.getVersion');
+    browser._isMac = version.userAgent.includes('Macintosh');
```

Comment:
Is this the remote-first world, i.e. now browser OS might be different from client OS?

Reply:
we've always had .connect. Yes